### PR TITLE
Feat/web app a11y round 2

### PR DESCRIPTION
**Pull request by bob-voorhoede**
### Further a11y improvements
- focus visibility and improved styling for all relevant elements
- clear indication when something is disabled
- improved document outline
- accessible svgs
- icons hidden from screen reader
- external link indicates that it opens a new window on screen reader
- proper landmarks
- submit button for top url input
- focus management after SPA routing

### CODE REVIEW

File: projects/web-app/src/components/app-shell/app-header.vue [hunk 1]
```diff
@@ -31,12 +36,20 @@ export default {
   left: 0;
   width: 100%;
   height: var(--header-height);
+}
+
+.app-header__left {
```
Comment (vvandijck) [inline projects/web-app/src/components/app-shell/app-header.vue]: I think `.app-header__section` or `.app-header__column` with a `--left` or `--right` modifier class is more BEM-like I think.

File: projects/web-app/vue.config.js [hunk 2]
```diff
@@ -11,7 +11,14 @@ module.exports = {
       .loader('vue-loader-v16')
       .end()
       .use('vue-svg-loader')
-      .loader('vue-svg-loader');
+      .loader('vue-svg-loader')
+      .options({
```
Comment (vvandijck) [inline projects/web-app/vue.config.js:15]: This did work after we discussed other options? 🥇

Comment (bob-voorhoede) [reply projects/web-app/vue.config.js:15]: I found it in an obscure section of the readme, which wasn't necessarily about svgo options

File: projects/web-app/src/components/app-shell/app-header.vue (hunk 1 above)
Comment (bob-voorhoede) [reply projects/web-app/src/components/app-shell/app-header.vue]: 👌

**Review by vvandijck — APPROVED** [review]

**Pull request merged**